src/components/molecules/Visualizer/Widget/index.tsx

```diff
@@ -36,6 +44,12 @@ export default function WidgetComponent<PP = any, SP = any>({
   widget,
   extended,
   pluginBaseUrl,
+  pluginModalContainer,
+  shownPluginModalInfo,
+  showPluginModal,
+  pluginPopupContainer,
+  shownPluginPopupInfo,
+  showPluginPopup,
```

Comment:
Add them to infobox block (`Visualizer/Infobox/Block`) also. There are many props, so consider using a rest spread operator: `<Hoge {...props} />`

```tsx
export default function WidgetComponent<PP = any, SP = any>({
  widget,
  extended,
  ...props,
```

Reply:
Updated.